### DISCUSSION TITLE
The same ignored version is suggested as part of a larger merge request

### DISCUSSION THREAD

**Discussion by nielsbasjes** [renovatebot/renovate#25632] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

The org.apache.maven.plugins:maven-artifact-plugin version 3.5.0 has a [problem](https://issues.apache.org/jira/browse/MARTIFACT-54) so I rejected the Renovate provided update https://github.com/nielsbasjes/yauaa/pull/1215

Renovate [responded](https://github.com/nielsbasjes/yauaa/pull/1215#issuecomment-1788901609) with the desired:

    Renovate will ignore this update (3.5.0). You will get a PR once a newer version is released.

Yet I noticed today that in a different merge request in the same project this same version is suggested again https://github.com/nielsbasjes/yauaa/pull/1219

![image](https://github.com/renovatebot/renovate/assets/240096/5e781a15-93ab-4948-94b8-e9e21dad5eb1)

### Relevant debug logs

The log is too big to paste in here
https://developer.mend.io/github/nielsbasjes/yauaa/-/job/018ba85f-98ee-7d04-8d9c-0cb0b2c07f9c

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Are the branch names the same?

**Reply by rarkins** [reply, marked as answer]
Renovate's ignores are based on the branch name + PR title combination. So if the title changes into a group title then it  is no longer ignored by renovate. Not a bug but more of a known limitation

**Reply by nielsbasjes** [reply]
In both cases it is the same branch name ( renovate/maven_build_plugins ) and the title IS different...

**Reply by nielsbasjes** [reply]
Why it the branch + title used instead of artifact id + version? It seems counterintuitive to me.

**Reply by rarkins** [reply]
Because renovate is stateless and uses the data on GitHub as its state. The PR is the state.
